conda/recipes/libcugraph/meta.yaml

```
@@ -116,14 +159,22 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
       run:
```

We're replacing the use of `cuda_spec` everywhere. Instead, we want to pin `cuda-version` in `host` and pin a compatible major version of `cuda-version` in `run` like shown below. Then remove the `{{ cuda_spec }}` pinnings and also remove the variable declaration. This probably applies several times in this PR / repo, please keep an eye out for that.
```suggestion
      host:
        - cuda-version ={{ cuda_version }}
      run:
        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```